.travis.yml

```diff
@@ -1,12 +1,14 @@
+sudo: false
 language: python
+cache: pip
```

Comment:
A bit tricky, but if we use the `install:` stanza along with the caching stanza, it won't work. See [here](https://github.com/edx/ecommerce/blob/master/.travis.yml#L6-L10).